src/pages/wallet-overview/wallet-overview.ts

```diff
@@ -765,41 +750,38 @@ export class WalletOverviewPage {
   /**
    * Show the distribution chart. False when chart could not be drawn
    */
-  displayChart(): boolean {
-    if (this.currenciesForDoughnutCanvasLabels === undefined || 
-        this.doughnutCanvas === undefined) {
-      return false;
-    }
-    this.doughnutChart = new Chart(this.doughnutCanvas.nativeElement, {
-      type: "doughnut",
-      data: {
-        datasets: [{
-          data: this.currenciesForDoughnutCanvas,
-          backgroundColor: [
-            "#064C70",
-            "#1B79A9"
-          ]
-        }],
-        labels: this.currenciesForDoughnutCanvasLabels
-      },
-      options: {
-        legend: {
-          display: false
+  displayChart(): void {
+    if (this.currenciesForDoughnutCanvasLabels !== undefined || this.doughnutCanvas !== undefined) {
+      this.doughnutChart = new Chart(this.doughnutCanvas.nativeElement, {
+        type: "doughnut",
+        data: {
+          datasets: [{
+            data: this.currenciesForDoughnutCanvas,
+            backgroundColor: [
+              "#064C70",
+              "#1B79A9"
+            ]
+          }],
+          labels: this.currenciesForDoughnutCanvasLabels
         },
-        legendCallback: function(chart) {
-          let text = [];
-          for (let i= 0; i < chart.data.datasets[0].data.length; i++) {
-              text.push({backgroundColor: chart.data.datasets[0].backgroundColor[i],
-                         label: chart.data.labels[i],
-                         data: chart.data.datasets[0].data[i]});
+        options: {
+          legend: {
+            display: false
+          },
+          legendCallback: function(chart) {
```

Comment:
For consistency I think it would be better to define the function using the `(chart) => { .... }` syntax. The only exception on this if we want to have the `this` shop of the function caller. This does not appear to be the case here?